common-custom-user-data-maven-extension/src/main/resources/gradle-enterprise.sample.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+
+<!-- Minimal gradle-enterprise.xml configuration file for Maven.
+
+     This file specifies only the URL of the Gradle Enterprise server, assuming that the rest of the Gradle Enterprise
+     configuration is provided programmatically in the extension.
+-->
+
+<gradleEnterprise
+    xmlns="https://www.gradle.com/gradle-enterprise-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="https://www.gradle.com/gradle-enterprise-maven https://www.gradle.com/schema/gradle-enterprise-maven.xsd">
+  <server>
+    <url>https://enterprise-samples.gradle.com</url>
```

Comment:
I think we used https://my-gradle-enterprise-server.com in other places. Can you double check and align?

Reply:
Actually we use https://your-gradle-enterprise-server.com. I'll make sure this is consistent.